cmd/cover.go

```diff
@@ -47,8 +49,11 @@ goc cover --center=http://127.0.0.1:7777 --target=/path/to/target --mode=atomic
 		if mode != "set" && mode != "count" && mode != "atomic" {
 			log.Fatalf("unknown -mode %v", mode)
 		}
-
-		cover.Execute(buildFlags, "", target, mode, agentPort, center)
+		_, p, err := net.SplitHostPort(agentPort)
```

Comment:
this is wrong place to modify since the `goc cover` command is hidden, user shoule not be able to use it.